### ISSUE TITLE
Niddler error 

### ISSUE THREAD

**Issue by k00na** [icapps/flutter-template#26]
After cloning the repo, running "flutter packages get" I tried to run the app but I got this error: 

```
lib/niddler.dart:13:3: Error: Getter not found: 'NiddlerInjector'.
  NiddlerInjector.install(niddler);
  ^^^^^^^^^^^^^^^
```
To fix this I had to replace this line:

`NiddlerInjector.install(niddler);`

with 

`niddler.install()`

**Comment by vanlooverenkoen** [comment]
Indeed this needs to be updated. Thanks.

**Comment by vanlooverenkoen** [comment]
fixed by #27
